apps/framework-docs/src/pages/building/capture-data.mdx

```diff
@@ -0,0 +1,55 @@
+import { Callout } from 'nextra/components'
+import { Tabs } from 'nextra/components'
+
+# Capture data
+MooseJS makes it easy to capture data from your user facing applications by providing typed SDKs which are automatically generated from your data models. 
+
+## From MooseJS SDKs
+<Callout type="info" emoji="ℹ️">
+  We currently support Typescript SDKs. Support for other languages is coming soon.
+</Callout>
+
+You can develop your data service right along side your user facing applications. MooseJS will automatically generate a Typescript SDK as you build your service locally and make them available for you to link to your application. To link a data service's SDK to your application, simply run the following command:
+
+```txt filename="Terminal"
+// From your application's root directory or wherever you run npm install
+npm link <project-name>-sdk 
+```
+
+## Through Ingestion Points
```

Comment:
For this example we should have the `prisma` file next to it that is used to validate and ingest the data